module/core/src/Application/Model/Type/LanguageConfigurationFormTypeModel.php

```diff
@@ -25,13 +25,13 @@ class LanguageConfigurationFormTypeModel
      *
      * @CoreAssert\LanguageCodeConstraint();
      */
-    public $code;
+    public Language $code;
```

Comment:
You need to use ?Language, this field can be null.
Symfony form set value on model class before validation. When value is null php 7.4 will throw and error becous of type restriction.

Also you need set initial null value in constructor or on property.